frontend/src/app/superadmin/users/users.component.ts

```diff
@@ -216,10 +216,20 @@ export class UsersComponent implements OnInit {
   }
 
   selectWorkspace(ws: IdRoleData, role: string): void {
-    if (ws.role === role) {
-      ws.role = '';
-    } else {
-      ws.role = role;
+    if (role === 'RW') {
```

Comment:
Hier sagt die IDE "ducplicate code fragment". Der gleiche Code ist in workspaces.component.ts. Aber ist vermutlich zu aufwändig das zu bereinigen. Von daher schaus dir an, wenn du willst. Lehne ich deswegen nicht ab.